litestar/types/builtin_types.py

```diff
@@ -23,3 +24,4 @@
 
 UnionTypes = {UnionType, Union}
 TypedDictClass: TypeAlias = Type[_TypedDictMeta]
+EmptyDict: TypeAlias = Dict[Never, Never]
```

Comment:
Its probably not worth adding a new library type just to type those tests - I'd be happy enough with `dict[Any, Any]` for them.

Reply:
Well, we don't have a documented standard on adding new types to the library, however if we did this use case would fall short of it IMO.

Reply:
Got it, will inline it